swag-chart/values.yaml

```diff
@@ -0,0 +1,171 @@
+  deploymentName: inventory
+  serviceName: inventory-service
+  app: inventory-app
+  image: aypas/inventory
+  tag: latest
+  memoryLimit: 256Mi
+  cpuLimit: 500m
+  replicas: 1
+  containerPort: 8100
+
+commerce:
+  baseURL: /commerce/
+  deploymentName: commerce
+  serviceName: commerce-service
+  app: commerce-app
+  image: aypas/commerce
+  tag: latest
+  memoryLimit: 256Mi
+  cpuLimit: 500m
+  replicas: 1
+  containerPort: 8200
+
+config:
+  deploymentName: config
+  serviceName: config-service
+  app: config-app
+  image: aypas/config
+  tag: latest
+  memoryLimit: 256Mi
+  cpuLimit: 500m
+  replicas: 1
+  containerPort: 8888
+
+eureka:
+  deploymentName: eureka
+  serviceName: eureka-service
+  app: eureka-app
+  image: aypas/eureka-server
+  tag: latest
+  memoryLimit: 256Mi
+  cpuLimit: 500m
+  replicas: 1
+  containerPort: 9000
+
+gateway:
+  deploymentName: gateway
+  serviceName: gateway-service
+  app: gateway-app
+  image: aypas/spring-cloud-gateway
+  tag: latest
+  memoryLimit: 256Mi
+  cpuLimit: 500m
+  replicas: 1
+  containerPort: 9001
+
+envVars:
+  emailHost:
+    key: email.host
+    value: gmail.com
+  emailPort:
+    key: email.port
+    value: 8080
+  emailUsername:
+    key: email.username
+    value: some_name
+  emailPassword:
+    key: email.Password
+    value: somePass
+  awsAccessID:
+    key: aws.accessKeyId
+    value: some_access_key_id
+  awsSecret:
+    key: aws.secretAccessKey
+    value: some_access_key
+
+ingress:
+  name: swag-ingress
+  useIngress: true
+
+replicaCount: 1
+
+image:
+  repository: nginx
+  pullPolicy: IfNotPresent
+  # Overrides the image tag whose default is the chart appVersion.
+  tag: ""
+
+imagePullSecrets: []
+nameOverride: ""
+fullnameOverride: ""
+
+serviceAccount:
+  # Specifies whether a service account should be created
+  create: true
+  # Annotations to add to the service account
+  annotations: {}
+  # The name of the service account to use.
+  # If not set and create is true, a name is generated using the fullname template
+  name: ""
+
+podAnnotations: {}
+
+podSecurityContext: {}
+  # fsGroup: 2000
+
+securityContext: {}
+  # capabilities:
+  #   drop:
+  #   - ALL
+  # readOnlyRootFilesystem: true
+  # runAsNonRoot: true
+  # runAsUser: 1000
+
+service:
+  type: ClusterIP
+  port: 80
+
+resources: {}
+  # We usually recommend not to specify default resources and to leave this as a conscious
+  # choice for the user. This also increases chances charts run on environments with little
+  # resources, such as Minikube. If you do want to specify resources, uncomment the following
+  # lines, adjust them as necessary, and remove the curly braces after 'resources:'.
+  # limits:
+  #   cpu: 100m
+  #   memory: 128Mi
+  # requests:
+  #   cpu: 100m
+  #   memory: 128Mi
+
+autoscaling:
+  enabled: false
+  minReplicas: 1
+  maxReplicas: 100
+  targetCPUUtilizationPercentage: 80
+  # targetMemoryUtilizationPercentage: 80
+
+nodeSelector: {}
+
+tolerations: []
+
+affinity: {}
```

Comment:
we'd like to take out or refactor the parts of this file that won't be relevant. e.g. i'm pretty sure service: and resources: here aren't being used.